tox.ini

```diff
@@ -14,6 +14,10 @@ minversion = 3.21
 
 [testenv]
 description = run the unit tests with pytest under {basepython}
+passenv =
+    ANDROID_DATA
+    ANDROID_ROOT
+    PYTHONPATH
```

Comment:
I guess we don't need it anymore

Reply:
Still needed when we're running the test suite on Android. Though PYTHONPATH is not needed.